public/docs/_examples/template-syntax/dart/lib/app_component.dart

```diff
@@ -0,0 +1,122 @@
+library template_syntax.app_component;
+
+import 'dart:html';
+import 'dart:convert';
+
+import 'package:angular2/angular2.dart';
+import 'package:template_syntax/hero.dart';
+import 'package:template_syntax/hero_detail_component.dart';
+import 'package:template_syntax/click_me_component.dart';
+import 'package:template_syntax/loop_back_component.dart';
+import 'package:template_syntax/key_up_components.dart';
+import 'package:template_syntax/little_hero.dart';
+
+List<Hero> _MockHeroes = [
+  new Hero('Hercules',
+      lastName: 'Son of Zeus',
+      birthdate: new DateTime(1970, 1, 25),
+      url: 'http://www.imdb.com/title/tt0065832/',
+      rate: 325),
+  new Hero('eenie', lastName: 'toe'),
+  new Hero('Meanie', lastName: 'Toe'),
+  new Hero('Miny', lastName: 'Toe'),
+  new Hero('Moe', lastName: 'Toe')
+];
+
+enum _Color { Red, Green, Blue }
+
+@Component(selector: 'my-app')
+@View(
+    templateUrl: 'my-app.html',
+    directives: const [
+      CORE_DIRECTIVES,
+      FORM_DIRECTIVES,
+      HeroDetailComponent,
+      ClickMeComponent,
+      LoopBackComponent,
+      KeyUpComponent,
+      KeyUpComponentV2,
+      KeyUpComponentV3,
+      KeyUpComponentV4,
+      LittleHeroComponent,
+    ])
+class AppComponent {
+  List<Hero> heroes = _MockHeroes;
```

Comment:
Just put the collection inline here.

Since it seems the goal is to modify the list anyway.